crates/ibc/src/core/ics02_client/handler/update_client.rs

```diff
@@ -311,6 +341,89 @@ mod tests {
         assert_eq!(client_state.latest_height(), latest_header_height);
     }
 
+    #[test]
+    fn test_update_synthetic_tendermint_client_validator_change_fail() {
+        let client_id = ClientId::new(tm_client_type(), 0).unwrap();
+        let client_height = Height::new(1, 20).unwrap();
+        let chain_id_b = ChainId::new("mockgaiaB", 1).unwrap();
+
+        let ctx = MockContextConfig::builder()
+            .host_id(ChainId::new("mockgaiaA", 1).unwrap())
+            .host_type(HostType::Mock)
+            .latest_height(Height::new(1, 1).unwrap())
+            .max_history_size(5)
```

Comment:
We can remove `.host_type()` and `.max_history_size()` given that we have access to these through default derivations.